react.html

```diff
@@ -70,10 +70,7 @@
   	  		strikes: this.state.strikes + 100
   	  	});
         if (this.state.strikes==1000) {
-          clearInterval(this.interval);
-          this.setState({
-          strikes: this.state.strikes + 100
-        });
+          (this.setState({strikes:0}));
```

Comment:
Looks right, but you don't need parentheses around the whole line. Indentation is also a bit weird in this section